RAP/rap.qmd

```diff
@@ -1162,4 +1192,4 @@ git clone YOUR_URL_HERE
 
 - Make use of git and version control in your team projects regularly. Like learning anything new, putting it into practice regularly is the best way to become confident in using it.  
 
-Please refer to the other links on the [learning resources](l+d.html#git) page to learn more about how to use git in practice.
+Please refer to the other links on the [learning resources](../learning-development/learning-development.html#git) page to learn more about how to use git in practice.
```

Comment:
This link doesn't work? Should it just go straight to the git page? (Tbh I suspect the old link didn't work either so it's not a new break...)